bucket/v-weekly.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "2021.45",
+    "description": "The V programming language (weekly release channel)",
+    "homepage": "https://vlang.io",
+    "license": "MIT",
+    "notes": [
+        "V requires a C compiler on Windows. You can use one of MinGW-w64, llvm-mingw, MSVC.",
```

Comment:
V also can download TCC
> V downloads TCC automatically

Reply:
Thank you for the info. I would be grateful if you can also provide me the source (I assume it's https://github.com/vlang/v#readme).

I think we can remove `V requires a C compiler...` and only keep the description about `Windows SDK`. Is that right?